src/tests/ftest/dfuse/daos_build.py

```diff
@@ -135,9 +135,11 @@ def run_build_test(self, cache_mode, intercept=False, run_on_vms=False):
 
         # Note that run_on_vms does not tell ftest where to run, this should be set according to
         # the test tags so the test can run with appropriate settings.
+        remote_env = {}
         if run_on_vms:
             dfuse_namespace = dfuse_namespace = "/run/dfuse_vm/*"
             build_jobs = 6 * 2
+            remote_env['D_IL_MAX_EQ'] = '6'
```

Comment:
Could this not be set at L199-203 along with the rest of the intercept options.

Reply:
i thought we want this only for vms and IL testing; but if you want to do this for all IL testing with this, then im fine to do that.

Reply:
Ah, I see the difference.  In practice they're the same thing, threre's only one variant of this test that's run on VMs and that's the IL variant.

Reply:
yea but i was thinking what if we add more variants on HW later? probably want those to run with default setting for that one.

Reply:
i would suggest to leave it like this for now.